apps/web/i18n/lang/en.json

```
@@ -222,6 +222,7 @@
       "createAccountLinkLabel": "Sign up now!",
       "forgotPasswordLabel": "Forgotten your password?",
       "heading": "Log in to your account",
+      "headingSuccess": "Reset password success",
```

```suggestion
      "title": "Password reset",
```
Also, this is in the wrong section. Should be in `resetPassword` instead of `auth`.